tests/Guzzle5ExceptionTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Http\Adapter\Tests;
+
+use GuzzleHttp\Exception as GuzzleExceptions;
+use Http\Adapter\Guzzle5HttpAdapter;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class Guzzle5ExceptionTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetException()
+    {
+        $request = $this->getMock('Psr\Http\Message\RequestInterface');
```

Comment:
There is nothing bad with simple mocks. For example with a simple return value check, but this seems like an overkill to me.